app-config.yaml

```diff
@@ -61,7 +61,7 @@ proxy:
     headers:
       Content-Type: 'application/json'
       accept: 'application/json'
-      # Authorization: ${PARODOS_AUTH_KEY}
+      Authorization: ${PARODOS_AUTH_KEY}
```

Comment:
Why is that change needed?
We should not use this anymore since we gather authentication at runtime.